test/Npgsql.Tests/Types/MiscTypeTests.cs

```diff
@@ -263,6 +263,27 @@ public void Jsonb()
             }
         }
 
+        [Test]
+        public void JsonbBytesRoundtrip()
+        {
+            using (var conn = OpenConnection())
+            using (var cmd = new NpgsqlCommand("SELECT @p", conn))
+            {
+                TestUtil.MinimumPgVersion(conn, "9.4.0", "JSONB data type not yet introduced");
+                var sb = new StringBuilder();
+                sb.Append(@"{""Key"": """);
```

Comment:
Maybe just a simple interpolated $ string?

Reply:
Just a note: the 2nd append has an integer argument, which makes it a repeat (of > 8k characters). This was mimicked from the test above it. Using interpolation, it would be allocating an 8k string for the contents, then allocating a slightly larger string for the interpolation. Probably does not make a difference either way, but I think that is why stringbuilder was used instead.

Reply:
Well, it's just a test so we don't really care... But it's not very important either way, do it any way you like.